src/Sylius/Bundle/InstallerBundle/Checker/CommandDirectoryChecker.php

```diff
@@ -0,0 +1,158 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\InstallerBundle\Checker;
+
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Debug\Exception\ContextErrorException;
+use Symfony\Component\Filesystem\Exception\IOException;
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\Finder\Iterator\RecursiveDirectoryIterator;
+use Symfony\Component\Security\Core\Exception\AccessDeniedException;
+
+/**
+ * @author Mateusz Zalewski <mateusz.zalewski@lakion.com>
+ */
+class CommandDirectoryChecker
```

Comment:
You're bloody hell right! I just couldn't came out with more proper name, but I would definitely turn on my creativity in nearest future and push PR with some pretty good name for this service ;)